adala/environments/kafka.py

```diff
@@ -61,12 +61,12 @@ async def message_receiver(self, consumer: AIOKafkaConsumer, timeout: int = 3):
                 try:
                     # Wait for the next message with a timeout
                     msg = await asyncio.wait_for(consumer.getone(), timeout=timeout)
-                    # print_text(f"Received message: {msg.value}")
+                    print_text(f"Received message: {msg.value}")
```

Comment:
If we're going to keep these around lets make it more clear where the log is coming from - because we have consumers in so many different parts of the code its not clear when looking at logs

Reply:
oops, this was for testing, will revert